common/changes/@microsoft/rush/stekycz-git-hooks-exec_2023-01-05-10-36.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "[rush] Create custom git hooks instead of copy",
```

Comment:
```suggestion
      "comment": "Generate scripts in the Git hooks folder referring to the actual hook implementations in-place in the Rush `common/git-hooks/` folder instead of copying the scripts to the Git hooks folder.",
```